01_fib/sirigithub/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+//function returns absolute value of an interger
+func abs(x int) int {
```

Comment:
the function can be taken from math lib. You're using it just once so casting to `int` should not be very painful. Can stay as is if you prefer.